EOLib/Domain/Character/CharacterActions.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using AutomaticTypeMapper;
```

Comment:
Looks like nothing actually changed here, please reset to the content of the master branch